pkg/ddl/table.go

```diff
@@ -308,14 +308,16 @@ func onCreateView(d *ddlCtx, t *meta.Meta, job *model.Job) (ver int64, _ error)
 	schemaID := job.SchemaID
 	tbInfo := &model.TableInfo{}
 	var orReplace bool
-	var oldTbInfoID int64
-	if err := job.DecodeArgs(tbInfo, &orReplace, &oldTbInfoID); err != nil {
+	var placeholder any // oldTblInfoID
+	if err := job.DecodeArgs(tbInfo, &orReplace, &placeholder); err != nil {
 		// Invalid arguments, cancel this job.
 		job.State = model.JobStateCancelled
 		return ver, errors.Trace(err)
 	}
 	tbInfo.State = model.StateNone
-	err := checkTableNotExists(d, t, schemaID, tbInfo.Name.L)
+
+	oldTableID, err := findOldTableID(d, t, schemaID, tbInfo.Name.L)
```

Comment:
Can you double check that `getGeneralJob` will not return the second running CREATE OR REPLACE VIEW job if the one is running?

Reply:
https://github.com/pingcap/tidb/pull/53720/commits/c52830e6a04a744499545d1f2e0fd4f322643f6d